src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,5 +1,39 @@
 import React from 'react';
+import { Post } from '../../types/Post';
+import { UserInfo } from '../UserInfo';
+import { CommentList } from '../CommentList';
 
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
-);
+type Props = {
+  post: Post;
+};
+
+export const PostInfo: React.FC<Props> = ({ post }) => {
+  const {
+    title,
+    body,
+    user,
+    comments,
+  } = post;
+
+  return (
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">{title}</h3>
+
+        <p>
+          {' Posted by  '}
+
+          {user && <UserInfo user={user} />}
+        </p>
```

Comment:
it will be better to add check for the whole line and show message only is `user` exists
```suggestion
        {user && (
          <p>
            {' Posted by  '}
            <UserInfo user={user} />
          </p>
        )}
```